### PR TITLE
fix: fixing incorrect course keys

### PR DESCRIPTION
**Pull request by kiram15**
Our PM made a mistake when fetching the course keys, here are the updated ones along with error handling.

### CODE REVIEW

File: course_discovery/apps/course_metadata/management/commands/backfill_enterprise_inclusion.py [hunk 1]
```diff
@@ -1,8 +1,13 @@
+import logging
+from ast import Or
```
Comment (Ali-D-Akbar) [inline course_discovery/apps/course_metadata/management/commands/backfill_enterprise_inclusion.py]: What is this import used for?

**Review by Ali-D-Akbar — CHANGES_REQUESTED** [review]
As I said in the previous PR, rather than using the constant file, it may be better to use a config model to store the file through django admin. In my suggested approach, you wouldn't have to create another PR to update the course keys data.

This way, we'll need another PR to remove the constants file.

**Comment by mphilbrick211** [comment]
@azanbinzahid would you mind taking a look at these changes? Thank you!

**Review by azanbinzahid — APPROVED** [review]

**Comment by DawoudSheraz** [comment]
@kiram15 Hi, is this still needed? Thanks

**Comment by mphilbrick211** [comment]
Hi @kiram15 - just following up on this. Please let us know if you plan to proceed with this pull request. Thanks!

**Pull request closed without merging**